includes/functions_import.php

```diff
@@ -1065,21 +1080,49 @@ function tsml_import_sanitize_meetings($meetings, $data_source_url = null, $data
             }
         }
 
-        // track group values
-        if (isset($meetings[$i]['group']) && !empty($meetings[$i]['group'])) {
+        // sanitize timezone if present
+        if (!empty($meetings[$i]['timezone'])) {
+            if (!in_array($meetings[$i]['timezone'], $all_timezones)) {
+                // make an attempt to find timezone
+                $timezone = str_replace(' ', '_', $meetings[$i]['timezone']);
+                $matches = array_values(array_filter($all_timezones, function($tz) use ($timezone) {
+                    return false !== stripos($tz, $timezone);
+                }));
+                if (count($matches)) {
```

Comment:
maybe we should say `if (count($matches) === 1) {` to avoid matching the wrong one

Reply:
My thinking was if we find more than one match, we pick the first. You think if someone puts something so vague it matches multiple values (count > 1), it should be discarded?

Reply:
yah, it seems dangerous to assume we know if there are several

what if someone put america for example. we would assume america/adak

Reply:
@joshreisner FYI, check the latest. I made a meatier `tsml_timezone_parse()` function that's a little large, but only accepts 1 match case, and then if no matches, will check some canned aliases. Right now it's just US aliases, but could be expanded if we want to torture someone:

https://www.timeanddate.com/time/zones/

Reply:
thanks! i like that it's more deterministic now